packages/core/src/features/telemetry/renderer/telemetry-decorator-for-show-details.injectable.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, createInstantiationTargetDecorator, instantiationDecoratorToken } from "@ogre-tools/injectable";
+import { parseKubeApi } from "../../../common/k8s-api/kube-api-parse";
+import showDetailsInjectable from "../../../renderer/components/kube-detail-params/show-details.injectable";
+import emitTelemetryInjectable from "./emit-telemetry.injectable";
+
+const telemetryDecoratorForShowDetailsInjectable = getInjectable({
+  id: "telemetry-decorator-for-show-details",
+  instantiate: (diForDecorator) => createInstantiationTargetDecorator({
+    target: showDetailsInjectable,
+    decorate: (instantiate) => (di) => {
+      const emitTelemetry = diForDecorator.inject(emitTelemetryInjectable);
+      const showDetails = instantiate(di);
+
+      return (...args) => {
+        emitTelemetry({
+          action: showDetailsInjectable.id,
+          params: {
+            kind: (() => {
+              try {
+                return parseKubeApi(args[0] || "").resource;
+              } catch {
+                return "";
```

Comment:
Nit: this is change in behaviour, previously errors where made known as part of "non-basic" telemetry.

Not accounting for if that's good or bad, commit name should indicate changes in behaviour.

Reply:
This was restored in later commit, I suppose.

Reply:
Moved and squashed some commits to make this all part of one commit